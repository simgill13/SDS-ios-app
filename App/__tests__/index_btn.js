
import 'react-native';
import React from 'react';
// import Ourchat from '../components/ourchat';
import renderer from 'react-test-renderer';
// import store from '../store';




import Btn from '../components/btn';
test('LOGIN', () => {
  const tree = renderer.create(
    <Btn />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
