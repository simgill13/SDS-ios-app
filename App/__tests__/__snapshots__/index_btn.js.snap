// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LOGIN 1`] = `
<View
  style={
    Object {
      "height": 60,
      "marginVertical": 10,
      "paddingHorizontal": 0,
    }
  }
>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hasTVPreferredFocus={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
        },
        Object {
          "alignItems": "center",
          "backgroundColor": "rgba(255,255,255,.3)",
          "borderRadius": 22,
          "flex": 1,
          "justifyContent": "center",
          "margin": 5,
          "padding": 10,
        },
      ]
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "alignItems": "center",
            "color": "#ffffff",
            "fontSize": 20,
            "fontWeight": "400",
            "justifyContent": "center",
          }
        }
      >
         
      </Text>
    </View>
  </View>
</View>
`;

exports[`testing btn component 1`] = `
<View
  style={
    Object {
      "height": 60,
      "marginVertical": 10,
      "paddingHorizontal": 0,
    }
  }
>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hasTVPreferredFocus={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
        },
        Object {
          "alignItems": "center",
          "backgroundColor": "rgba(255,255,255,.3)",
          "borderRadius": 22,
          "flex": 1,
          "justifyContent": "center",
          "margin": 5,
          "padding": 10,
        },
      ]
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "alignItems": "center",
            "color": "#ffffff",
            "fontSize": 20,
            "fontWeight": "400",
            "justifyContent": "center",
          }
        }
      >
         
      </Text>
    </View>
  </View>
</View>
`;
